kili/graphql_client.py

```diff
@@ -12,14 +12,30 @@
 import websocket
 from six.moves import urllib
 
+from . import __version__
+
+
+class GraphQLClientName:  # pylint: disable=too-few-public-methods
+    """GraphQL client name."""
+
+    SDK = "python-sdk"
+    CLI = "python-cli"
```

Comment:
the fact the cli is coded in Python is an implementation detail, so I would say `kili-cli`

Reply:
As there is (was) a node client, I think we can keep python